WalletWasabi.Fluent/ViewModels/Wallets/Home/History/HistoryItems/HistoryItemViewModelBase.cs

```diff
@@ -47,6 +52,21 @@ protected HistoryItemViewModelBase(int orderIndex, TransactionSummary transactio
 
 	public ICommand? ShowDetailsCommand { get; protected set; }
 
+	protected virtual ObservableCollection<HistoryItemViewModelBase> LoadChildren()
+	{
+		throw new NotSupportedException();
+	}
+
+	public virtual void Update(HistoryItemViewModelBase item)
+	{
+		OrderIndex = item.OrderIndex;
+		Date = item.Date;
+		DateString = item.DateString;
+		IsConfirmed = item.IsConfirmed;
+	}
+
+	public bool IsSimilar(HistoryItemViewModelBase item) => Id == item.Id;
```

Comment:
They are removed on master. You can do as well

Reply:
It got added accidentally after my last merge resolve conflict

Reply:
Removed